apps/backend/src/libs/modules/token/jwt.service.ts

```diff
@@ -0,0 +1,26 @@
+import { type BaseToken } from "./base-token.js";
+import { type JwtPayload } from "./libs/types/types.js";
+
+class JwtService {
```

Comment:
Why do we need this service at all? It does the same thing as BaseToken. We can only have BaseToken and that will be enough